.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: false
 contact_links:
-  - name: QHub Documentation 📖
-    url: https://docs.qhub.dev/en/stable/
-    about: Read QHub docs and development guide.
+  - name: Nebari Documentation 📖
+    url: https://docs.nebari.dev/en/stable/
+    about: Read Nebari docs and development guide.
   - name: GitHub discussions 💬
-    url: https://github.com/Quansight/qhub/discussions
+    url: https://github.com/Quansight/nebari/discussions
     about: Keep up to date with the community discussions
   - name: "(maintainers only) Blank issue"
-    url: https://github.com/Quansight/qhub/issues/new
+    url: https://github.com/Quansight/nebari/issues/new
     about: For maintainers only - should be used sparingly
```

Comment:
Should we update this @trallard or use the ones from the organization? I think they are globally present in all repos under nebari-dev right?